artemis-hawtio/artemis-plugin/src/main/webapp/plugin/js/artemisPlugin.js

```diff
@@ -231,6 +234,17 @@ var ARTEMIS = (function(ARTEMIS) {
          }
       });
 
+      workspace.subLevelTabs.push({
```

Comment:
If the intent is not to delete a divert as per reply to previous comment, then this should not be added